Misc/NEWS.d/next/Library/2021-07-21-22-57-51.bpo-19217.vm-cr-.rst

```diff
@@ -0,0 +1,2 @@
+Optimize :meth:`~unittest.TestCase.assertEqual` method for long sequences of varied
```

Comment:
Mention what observable change users will notice.  Faster assertion reporting, but also presumably differently formatted diffs in formerly slow cases?

Reply:
Ok, I updated it in fc752cf40b752a28781e67b457de0d31f54dee14. Let me know if you have any thoughts.